test/Feature/DoubleFree.c

```diff
@@ -1,12 +1,14 @@
 // RUN: %clang %s -emit-llvm %O0opt -c -o %t1.bc
 // RUN: rm -rf %t.klee-out
-// RUN: %klee --output-dir=%t.klee-out %t1.bc 2>&1 | FileCheck %s
+// RUN: %klee --output-dir=%t.klee-out --kdalloc %t1.bc 2>&1 | FileCheck %s
```

Comment:
With kdalloc being the default, this change should be superfluous.

Reply:
Yes, but I think it's important to add the flag, as otherwise the test fails without KDAlloc.  
As a general rule, we (should) explicitly add the flags which are required for tests to pass -- they act as documentation and make them less fragile to changing defaults.